modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -107,111 +107,434 @@ Status description:: error: data exception - unsupported value. The provided val
+
+// === 22N82
+
+// Status description:: error: data exception - input contains invalid characters. Input `{ $input }` contains invalid characters for `{ $context }`. Allowed characters are `{ $characterRangeList }`. To use these characters, the input needs to be quoted using backticks.
+
+// === 22N83
+
+// Status description:: error: data exception - input consists of too many components. Expected name to contain at most `{ $upper }` components separated by '.'.
+
+=== 22N84
+
+Status description:: error: data exception - string too long. Expected the string to be no more than `{ $upper }` characters long.
+
+=== 22N85
+
+Status description:: error: data exception - string too short. Expected the string to be at least `{ $lower }` characters long.
+
+=== 22N86
+
+Status description:: error: data exception - numeric range 0 disallowed. Expected a nonzero number.
+
+// === 22N87
+
+// Status description:: error: data exception - numeric range 0 or greater allowed. Expected a number that is zero or greater.
+
+// === 22N88
+
+// Status description:: error: data exception - not a valid CIDR IP.
+
+=== 22N89
+
+Status description:: error: data exception - new password cannot be the same as the old password. Expected the new password to be different from the old password.
+
+=== 22N90
+
+Status description:: error: data exception - property type unsupported in constraint. `{ $item }` is not supported in property type constraints.
+
+=== 22N91
+
+Status description:: error: data exception - cannot convert alias local to remote or remote to local. Failed to alter the specified database alias `{ $alias }`. Altering remote alias to a local alias or vice versa is not supported. Drop and recreate the alias instead.
+
+// === 22N92
+
+// Status description:: error: data exception - missing RETURN. This query requires a RETURN clause.
+
+// === 22N93
+
+// Status description:: error: data exception - missing YIELD. A required YIELD clause is missing.
+
+// === 22N94
+
+// Status description:: error: data exception - invalid YIELD *. 'YIELD *' is not supported in this context. Explicitly specify which columns to yield. 
+
+=== 22N95
+
+Status description:: error: data exception - parsing JSON exception. Invalid JSON input. Please check the format.
+
+=== 22N96
+
+Status description:: error: data exception - mapping JSON exception. Unable to map the JSON input. Please verify the structure.
+
+=== 22N97
+
+Status description:: error: data exception - unexpected struct tag. Unexpected struct tag: `{ $tagStringAsHex }`.
+
+=== 22N98
+
+Status description:: error: data exception - wrong first field during deserialization. Unable to deserialize request. Expected first field to be `{ %s }`, but was '{ %s }'.
+
+=== 22N99
+
+Status description:: error: data exception - wrong token during deserialization. Unable to deserialize request. Expected `{ %s }`, found `{ %s }`.
```

Comment:
```suggestion
Status description:: error: data exception - wrong token during deserialization. Unable to deserialize request. Expected `{ $token }`, found `{ $value }`.
```

The spreadsheet had not written out the parameter names for this one, I updated it to this now which is what they are called in the codebase.